yamllint/cli.py

```diff
@@ -84,11 +84,14 @@ def standard_color(problem, filename):
         return line
 
 
-def show_problems(problems, file, args_format):
+def show_problems(problems, file, args_format, quiet):
     max_level = 0
     first = True
 
     for problem in problems:
+        max_level = max(max_level, PROBLEM_LEVELS[problem.level])
```

Comment:
I'm curious: why did you move this line?

Is it to keep a non-zero return code if `--strict` is used in the same time as `--no-warnings`? (I think it's a good thing). Can you add a `test_run_no_warnings_and_strict()`? for that?

Reply:
I wanted to keep the value of the return code , even if I filter the output.

Added the extra test for checking both options given in the same run.

Reply:
:+1: